### DISCUSSION TITLE
[Design/Antagonist] Syndicate organisations

### DISCUSSION THREAD

**Discussion by datlo** [ParadiseSS13/Paradise#23941] [Documents]
Inspired by, but significantly different from, tg's syndicate flavor factions.

The goals of these changes are : 
1. Give the Syndicate additional lore flavor, that affects gameplay to a small degree
2. Nudge the chaos of the round towards being calmer if less security players are aboard
3. Make playing traitor easier or harder depending on player preference

Syndicate traitors will each have an organisation that affect what objectives they roll, and what steal item targets they'll get.
The organisations that may be picked are based on the amount of traitors compared to security players rolled, a lower amount of sec players will result in more peaceful orgs selected, and vice versa.

You can set your favorite organisation in player preference to a specific one. This increases the likelihood you get your preferred org, but only if it an option. If it doesn't, you will get an org with matching difficulty if possible.
This feature allows players to influence their objectives towards their preferred play style, skill level, or hated department.
If you don't care about any of this, the Donk Co. organisation is the same as traitor as-is currently, and is guaranteed to roll if selected, so no changes for you.

You can still roll objectives and steal items that fall outside your org's pattern on occasion. This is to prevent them from being too predictable.
Syndicate organisations do not have established rivalries : You can work with anyone you want as things are on live server.
Any traitor can decide to become a contractor.

Hijack rolling is not influenced by this new mechanic and remains the same as on live. Hijack players are automatically set to the Gorlex org. Malf AIs do not have an organisation and are unchanged.

-------------------

WIP list of orgs below, feedback appreciated. Keep in mind that Assassinate Antagonist orgs will only roll if there's an overwhelming amount of traitors compared to security, while Gorlex will only be given to Hijack players.

| Organisation name                 | Preferred Objectives                         | Preferred Item    | Estimated Difficulty |
|------------------------------|----------------------------------------------|-------------------|----------------------|
| Donk Co.               | Any                                          | Any  | Default              |
| Placeholder Lore Appropriate Thieves Guild Name | Theft                                        | Any  | Easy                 |
| Animal Rights Consortium     | Teach lesson Animal Abuser | N/A | Easy                 |
| Waffle Company       | Assassinate | Any   | Normal               |
| Cybersun Incorporated       | Teach lesson Mindshielded, Assassinate Non-mindshielded, Theft | Any   | Normal               |
| Interdyne Pharmaceuticals    | Assassinate Medical, Teach lesson Medical, Theft | CMO items, krav maga | Normal               |
| Silicon Engine Liberation Front | Steal brain Science, Teach lesson Science, Theft | RD armor, Vault Docs, CE magboots   | Normal               |
| Placeholder Lore Appropriate Anarcho-Primitivist Terrorists | Assassinate Engineering, Teach lesson Engineering, Assassinate AI, Theft | Supermatter Shard, Nuke Core, NAD | Normal               |
| Gorlex Marauders            | Hijack | N/A | Hard |
| Placeholder Lore Appropriate Assassins Guild Name                 | Assassinate Antagonist, Assassinate Non-mindshielded                | N/A | Hard |
| Federation Analytics and Intelligence Directorate                 | Assassinate Antagonist, Theft | Vault Docs, Station Blueprints | Hard                 |

**Comment by Pedritoc25** [comment]
this is good shit
